lib/routes/install.js

```diff
@@ -61,7 +61,7 @@ function searchMatchingGenerators(app, term, cb) {
       }));
     }, cb);
   }
-  got('yeoman.io/blacklist.json', {json: true})
+  got('http://yeoman.io/blacklist.json', {json: true})
```

Comment:
https://github.com/sindresorhus/got/releases/tag/v8.0.0

> [Breaking change] Uses HTTPS for protocol-less URLs (Previously it defaulted to http)

Currently `yeoman.io` doesn't seem to support HTTPS. https://yeoman.io